requirements/edx/base.txt

```diff
@@ -683,7 +683,7 @@ jsonschema==4.23.0
     # via
     #   drf-spectacular
     #   optimizely-sdk
-jsonschema-specifications==2023.12.1
+jsonschema-specifications==2024.10.1
```

Comment:
New version added support for `Python 3.13` [[CHANGELOG](https://github.com/python-jsonschema/jsonschema-specifications/releases/tag/v2024.10.1)]